Apps/US/HybridGP_US/app/src/Codeunits/GPCloudMigrationUS.Codeunit.al

```diff
@@ -73,17 +78,10 @@ codeunit 42004 "GP Cloud Migration US"
 
     internal procedure IsIRSFormsFeatureEnabled(): Boolean
     var
-#if not CLEAN25
         FeatureManagementFacade: Codeunit "Feature Management Facade";
-#endif
         IsEnabled: Boolean;
     begin
-        IsEnabled := true;
-
-#if not CLEAN25
         IsEnabled := FeatureManagementFacade.IsEnabled(IRSFormFeatureKeyIdTok);
```

Comment:
The IRSFormFeatureKeyIdTok is removed if CLEAN25. Either also remove the call, or leave the variable declaration unconditional.

Reply:
Leaving the variable and removed the condition.